src/index.css

```diff
@@ -0,0 +1,351 @@
+body {
+  background-color: #fff;
+  color: #1e1e1e;
+  font-size: 14px;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: 'Poppins', sans-serif;
+}
+
+a {
+  text-decoration: none;
+}
+
+.logo {
+  display: flex;
+  align-items: center;
+  margin-left: 100px;
+}
+
+.logo > p {
+  font-size: 25px;
+  color: rgb(66, 106, 66);
+  border: 3px solid white;
+  padding: 10%;
+  border-radius: 10px;
+}
+
+.privacy-policy > a {
+  font-size: 25px;
+  color: rgb(66, 106, 66);
+  border: 3px solid white;
+  padding: 10%;
+  border-radius: 10px;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  background-color: blanchedalmond;
+  align-items: center;
+  box-shadow: 5px 2px 8px 2px #424242;
+  padding: 20px 0;
+}
+
+.header-links {
+  font-family: 'Open Sans', sans-serif;
+  display: flex;
+  margin-right: 20px;
+  gap: 20px;
+  font-size: 19px;
+}
+
+.header-links > a {
+  color: #1e1e1e;
+  transition: 0.5s ease-in-out;
+}
+
+.fab {

```

Comment:
It would be best to add names that are easy to get from the HTML file for easy reading of other coders